README.md

```diff
@@ -36,21 +38,20 @@ The `GITHUB_TOKEN` secret is [automatically generated by GitHub](https://help.gi
 
 By default, you probably don’t need to configure `image-actions`. However, if you’d like to ignore specific file paths, or change image compression options, read on.
 
-Change the configuration options by adding a `.github/calibre/image-actions.yml` file:
+Change the configuration options by adding arguments to the action:

```

Comment:
Is there a documentation reference for action arguments that can be added to the readme here?

Reply:
I don't really think it's necessary, as that's explained by the following lines.